pynicotine/gtkgui/chatrooms.py

```diff
@@ -257,7 +257,13 @@ def join_room(self, msg):
 
         self.pages[msg.room] = tab = ChatRoom(self, msg.room, msg.users)
 
-        self.append_page(tab.container, msg.room, focus_callback=tab.on_focus, close_callback=tab.on_leave_room)
+        if msg.room == core.chatrooms.GLOBAL_ROOM_NAME:
+            is_global = True
```

Comment:
Assign `is_global = (msg.room == core.chatrooms.GLOBAL_ROOM_NAME)` before the if clause, then check `if is_global`